cloudbuild.yaml

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Builds Docker image with Ground APK using Cloud Build. Specifically, executes the following steps:
```

Comment:
Did I get this right?

Reply:
I missed to mention this that yaml file is submitted automatically by the github app and the build is  triggered. 
It loads (not builds) the docker image  (from the container registry) and then runs the mentioned steps.

Reply:
Ah, sorry I misunderstood. Updated, hope this is better.

Reply:
Thank you for your updates!!! It looks a lot better now